draft-private-access-tokens.md

```diff
@@ -745,14 +745,28 @@ such as TLS certificate pinning, to mitigate the risk of channel compromise.
 An attacker that can intermediate the channel between Client and Origin can
 observe a TokenChallenge, and can view a Token being presented for authentication
 to an Origin. Scoping the TokenChallenge nonce to the Client HTTP session prevents
-Tokens being collected in one session and then presented to the Origin in another. 
-Note that an Origin cannot distinguish between a connection to a single Client and 
+Tokens being collected in one session and then presented to the Origin in another.
+Note that an Origin cannot distinguish between a connection to a single Client and
 a connection to an attacker intermediating multiple Clients. Thus, it is possible for
-an attacker to collect and later present Tokens from multiple clients over the same 
+an attacker to collect and later present Tokens from multiple clients over the same
 Origin session.
 
 # Privacy Considerations {#privacy-considerations}
 
+## Client Token State and Origin Tracking
+
+Origins SHOULD NOT challenge clients for access tokens more than once in a given context,
```

Comment:
Might be worth specifying what counts as a context, specifically whether it's limited to the policy window.

Reply:
It's very specifically not a policy window—there can be many challenges in a window. 

I'd suggest maybe phrasing this all as: "Origins SHOULD only generate token challenges based on client action, such as when a user loads a website. Clients SHOULD ignore token challenges if an Origin tries to force the client to present tokens multiple times without any new client-initiated action."